### PR TITLE
clarify nav nodes

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/8166

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteScriptFromWeb.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Accept wildcard characters: False
 ### -IncludedLists
 
 An array of one or more lists. Each is identified by the list url.
+Note: Navigation nodes are not being currently exported.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteScriptFromWeb.md:71]:
```suggestion

Note: Currently, navigation nodes are not exported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestion.

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**